src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
That is not a solution, thats a half arsed fix. A proper fix needs to be rolled out. 

Regardless, I can see no valid use for duplicating the same list of colours twice. Why not use the same list in both places, it would make absolutely no difference.